todo-frontend/src/services/user.service.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+axios.interceptors.request.use(
+    (config) => {
+      let token = localStorage.getItem('token');
+      if (token != '') {
+        config.headers['Authorization'] = 'Bearer ' + token;
+      }
+  
+      return config;
+    }, 
+  
+    (error) => {
+      return Promise.reject(error);
+    }
+);
+
+const API = 'http://localhost:8000/api';
```

Comment:
zato bi trebalo izvuci u dotenv
sada kad bi hteo da izbildas projekat za produkciju, on bi ovde umesto localhost trebalo da ima nesto-drugo.com. i morao bi da menjas kod svaki put kad bildas da bi to radilo
1. morao bi da menjas to jer nije nikako parametrizovano, u smislu da kada bildas koristi jednu vrednost a kad radis lokalno neku drugu
2. morao bi da menjas na vise mesta jer nije izvuceno u neki vid configa koji stoji na jednom mestu